### PR TITLE
Hompage revamp

### PR DESCRIPTION
**Pull request by ebuka-nwabueze**
This branch was created from the orthoex landing. 
It consists of changes of the home page and about.

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components"
```
Comment (Ozoemenamsolomon) [inline src/components/Button.tsx]: Why not use CTA and extend it if need be instead of creating a new button?

File: src/components/styled/index.tsx [hunk 2]
```diff
@@ -34,12 +34,11 @@ export const ServicesCards = styled.div<{ minWidth?: string }>`
 `;
 
 export const PostCardsContainer = styled.div`
-	// TODO make the breakpoint variables
-	@media (min-width: 600px) {
-		display: grid;
-		grid-template-columns: repeat(auto-fit, minmax(336px, 1fr));
-		gap: 4rem;
-	}
+	display: grid;
```
Comment (Ozoemenamsolomon) [inline src/components/styled/index.tsx:37]: Why did you remove the media query?

File: src/components/InfoCard.tsx [hunk 3]
```diff
@@ -61,29 +70,46 @@ const StyledInfoCard = styled.div`
...
     & > div > img {
       border-radius: 0rem;
     }
   }
+`;
 
-`
-
-const ButtonStyled = styled.button`
+export const StyledButton = styled.button`
```
Comment (Ozoemenamsolomon) [inline src/components/InfoCard.tsx]: Unify your components. You created a button component and now you have another here or are they different?

File: src/components/CallButton.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        />
+      </svg>
+      Call
+    </StyledCallButton>
+  );
+};
+
+export default CallButton;
+
+const StyledCallButton = styled.button`
```
Comment (Ozoemenamsolomon) [inline src/components/CallButton.tsx]: And you have another button definition here

File: src/components/CallButton.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  position: fixed;
+  left: 0;
+  bottom: 2%;
+  z-index: 9999;
```
Comment (Ozoemenamsolomon) [inline src/components/CallButton.tsx]: why?

File: src/components/CallButton.tsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "styled-components";
+
+const CallButton = () => {
+  return (
+    <StyledCallButton >
```
Comment (Ozoemenamsolomon) [inline src/components/CallButton.tsx]: What happens onClick?

File: src/pages/about.tsx [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+import AboutInfo from "../components/About/AboutInfo";
+import { Container } from "@components/styled";
+
+function About() {
+  const [currentSelected, setcurrentSelected] = useState<string>("overview");
+
+  return (
+    <Container bg="#FAFAFA" paddingMultiplier={0}>
+      <StyledAbout>
+        <AboutInfo setCurrentSelected={setcurrentSelected} />
```
Comment (Ozoemenamsolomon) [inline src/pages/about.tsx]: Your implementation of the tabs works and it's beautiful. From a scalability POV, consider using this datastructure to define your data `
[
    {
        title:"Company overview",
        content: then your content here
    },
    {
        title:"Company overview",
        content: then your content here
    },
    {
        title:"Company overview",
        content: then your content here
    }
]`

File: src/components/Button.tsx (hunk 1 above)
Comment (ebuka-nwabueze) [reply src/components/Button.tsx]: component removed.

File: src/components/styled/index.tsx (hunk 2 above)
Comment (ebuka-nwabueze) [reply src/components/styled/index.tsx:37]: The same design dimensions were needed from small breakpoints. That is why i moved it out of the media query which started at 600px.

File: src/components/CallButton.tsx (hunk 5 above)
Comment (ebuka-nwabueze) [reply src/components/CallButton.tsx]: This button is the call button in the homepage on the mobile screen.

File: src/components/CallButton.tsx (hunk 6 above)
Comment (ebuka-nwabueze) [reply src/components/CallButton.tsx]: It's meant to direct calling to a phone number. 
If there is a phone number available, i can add that.

File: src/components/CallButton.tsx (hunk 4 above)
Comment (ebuka-nwabueze) [reply src/components/CallButton.tsx]: CTA component has been used now.

File: src/components/InfoCard.tsx (hunk 3 above)
Comment (ebuka-nwabueze) [reply src/components/InfoCard.tsx]: It has been removed.

File: src/components/CallButton.tsx [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+export default CallButton;
+
+const StyledCallButtonDiv = styled.div`
+position: fixed;
+left: 0;
+bottom: 2%;
+z-index: 10;
+
+& > button {
```
Comment (Ozoemenamsolomon) [inline src/components/CallButton.tsx]: Bro, you're repeating styling. Please take time and look into the CTA comp. It already has properties like bg. I'd recommend: remove these styling, just put the CTA first then start adjusting it where it might need adjustment

File: src/components/IconText.tsx [hunk 9]
```diff
@@ -14,7 +14,7 @@ const IconText: FC<{ icon: StaticImageData; text: string }> = ({
 		}}
 	>
 		<span>{text}</span>
-		<Image objectFit="contain" src={icon} />
+		<Image alt='image' objectFit="contain" src={icon} />
```
Comment (Ozoemenamsolomon) [inline src/components/IconText.tsx]: use a more descriptive alt or leave it out: in this case "icon" would be better

File: src/components/InfoCard.tsx [hunk 10]
```diff
@@ -61,29 +71,54 @@ const StyledInfoCard = styled.div`
...
-  padding: 1rem;
+  padding: 0rem;
   width: 100%;
-  border-radius: 0.2rem;
-  border: 0.09rem solid var(--oex-orange);
 
-  &:hover {
-    color: white ;
-    background-color: var(--oex-orange);
+  & > button {
```
Comment (Ozoemenamsolomon) [inline src/components/InfoCard.tsx]: you have a "white" prop why should this have a background of white again?

File: src/components/styled/index.tsx (hunk 2 above)
Comment (Ozoemenamsolomon) [reply src/components/styled/index.tsx:37]: Okay, cool! 👍🏽

File: src/pages/about.tsx [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+  values: valuesData
+}
+
+function About() {
+  const [currentSelected, setcurrentSelected] = useState<string>("overview");
+
+  return (
+    <Container bg="#FAFAFA" paddingMultiplier={0}>
+      <StyledAbout>
+        <AboutInfo data={AboutInfoData} setCurrentSelected={setcurrentSelected} />
```
Comment (Ozoemenamsolomon) [inline src/pages/about.tsx]: We need to review this together, if you use an array, then your state should be of type `number`

File: src/components/ProductVerticalCard.tsx [hunk 12]
```diff
@@ -10,7 +10,7 @@ const ProductVerticalCard: React.FC<VerticalType> = ({image,link, title}) => {
   return (
     <StyledProductVertical>
       <StyledImageWrapper>
-        <Image width="100" height="100" quality={100} objectFit="cover" layout="fill" src={image} />
+        <Image quality={100} objectFit="cover" layout="fill" src={image} alt="title" />
```
Comment (Ozoemenamsolomon) [inline src/components/ProductVerticalCard.tsx]: more descriptive alt: is it the `title-image` or...?

File: src/components/InfoCard.tsx [hunk 13]
```diff
@@ -61,29 +71,54 @@ const StyledInfoCard = styled.div`
...
-  background-color: white;
+const ButtonContainer = styled.div`
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+  justify-content: center;
+  width:100%;
   font: inherit;
-  padding: 1rem;
+  padding: 0rem;
```
Comment (Ozoemenamsolomon) [inline src/components/InfoCard.tsx:87]: then remove the padding

File: src/components/InfoCard.tsx (hunk 10 above)
Comment (Ozoemenamsolomon) [reply src/components/InfoCard.tsx]: the fact that you're styling the buttons inside their container shows you are just using the CTA just because it's a CTA, you can modify the CTA to fit your need

File: src/components/CallButton.tsx (hunk 8 above)
Comment (ebuka-nwabueze) [reply src/components/CallButton.tsx]: I have removed the background color and border color. I skipped that.

File: src/components/InfoCard.tsx (hunk 10 above)
Comment (ebuka-nwabueze) [reply src/components/InfoCard.tsx]: It has been moved into CTA. I created a custom style that inherits from CTA

File: src/components/IconText.tsx (hunk 9 above)
Comment (ebuka-nwabueze) [reply src/components/IconText.tsx]: It has been changed to icon

File: src/pages/about.tsx (hunk 11 above)
Comment (ebuka-nwabueze) [reply src/pages/about.tsx]: Ok. We can put it in the backlog for a later time.

File: src/components/ProductVerticalCard.tsx (hunk 12 above)
Comment (ebuka-nwabueze) [reply src/components/ProductVerticalCard.tsx]: it is the image title, i replaced with a dynamic value ${title} instead

File: src/components/InfoCard.tsx (hunk 13 above)
Comment (ebuka-nwabueze) [reply src/components/InfoCard.tsx:87]: Done

**Review by Ozoemenamsolomon — COMMENTED** [review]
| image  | image  |
|---|---|
| please use the already existing card component | the same ones used in this image, it has an optional title prop, this could have saved a lot of time  |
